internal/builder/builder.go

```
@@ -284,6 +290,54 @@ func (i *CIFuzzBuilder) BuildProcessWrapper() error {
 	return nil
 }
 
+func (i *CIFuzzBuilder) BuildListFuzzTestsTool() error {
+	var err error
+	err = i.Lock()
+	if err != nil {
+		return err
+	}
+	defer func() {
+		err = i.Unlock()
+		if err != nil {
+			log.Printf("error: %v", err)
+		}
+	}()
+
+	// Build tool in temp dir
+	tempDir, err := os.MkdirTemp("", "list-fuzz-tests-")
+	if err != nil {
+		return errors.WithStack(err)
+	}
+	defer fileutil.Cleanup(tempDir)
+
+	err = copy.Copy(filepath.Join(i.projectDir, "tools", "list-fuzz-tests"), tempDir)
```

This error is not checked